scripts/lib/svg-generation-fns.ts

```diff
@@ -147,17 +149,26 @@ export async function processSvg(symbolsSvg: string, fileName?: string) {
             console.log(`  - paths: ${Object.keys(paths).length}`)
             console.log(`  - refblocks: ${Object.keys(refblocks).length}`)
           } else if (!hasSourceFile && isReadyForGen) {
-            console.log(`Creating horz source file: ${outputPath}`)
+            console.log(
+              `Creating ${defaultOrientation} source file: ${outputPath}`,
+            )
             const content = getTsFileContentForSvgGroup(groupId, svgData as any)
             fs.writeFileSync(outputPath, content)
-            // Write the vert file
-            const vertOutputPath = `./symbols/${groupId}_vert.ts`
-            console.log(`Creating vert source file: ${vertOutputPath}`)
-            const contentVert = `import { rotateSymbol } from "drawing/rotateSymbol"\nimport ${groupId}_horz from "./${groupId}_horz"\n\nexport default rotateSymbol(${groupId}_horz)`
-            fs.writeFileSync(vertOutputPath, contentVert)
+
+            // Generate additional orientation files based on the symbol type
+            const orientations = isDiode ? ["left", "up", "down"] : ["vert"]
```

Comment:
don't introduce specific flags like this into general-purpose files. As we add symbols, people will keep adding flags here and this file would become unmaintainable.

Reply:
i took a quick look and would recommend this approach

![image](https://github.com/user-attachments/assets/375df7a0-c142-412c-80ab-8beaa976c627)

Reply:
I think this code will create 6 files ["horz" , "vert", "up","left" ,"down" , "right"] for each groupID !!

Reply:
my code was incomplete/untested, just something to hopefully get you started!

Can you remove the `isDiode` check in your code? Assume that if the `_right` orientation is defined, then all other directional rotations should be defined

Reply:
oh i see what you're saying. Can we ask the user with an additional interactive prompt?

Reply:
@seveibar 

![image](https://github.com/user-attachments/assets/60ab7c03-3f42-40c8-8283-a5cced263ca2)

Reply:
Yea that's perfect